src/commands/index.js

```diff
@@ -24,24 +24,24 @@ const pj = require('../../package.json');
 
 program
 .version(pj.version)
-.command('register --name <name> --url <url> --token <token>',
-  'Register collector with an instance of Refocus')
 .command('start --collectorName <name> --refocusUrl <url> --accessToken <token>',
   'Start collector')
 .command('stop --name <name>', 'Stop collector')
 .command('status --name <name>', 'Show current status of collector')
-.command('deregister --name <name>', 'Deregister collector');
+.command('deregister --name <name>', 'Deregister collector')
```

Comment:
`--collectorName`?

Reply:
If I will do this I need to change command files that is not part of story.

Reply:
So let me know what to do @iamigo